### PR TITLE
Adapt to subiquity's new /shutdown endpoint 

### PR DESCRIPTION
**Pull request by jpnurmi**
Solves: #224

### CODE REVIEW

**Comment by jpnurmi** [comment]
Depends on https://github.com/canonical/subiquity/pull/1029

File: packages/subiquity_client/lib/subiquity_client.dart [hunk 1]
```diff
@@ -231,8 +231,19 @@ class SubiquityClient {
...
+        Uri.http('localhost', 'shutdown',
+            {'mode': '"REBOOT"', 'immediate': '$immediate'}));
+    await _client.write(request);
+  }
+
+  Future<void> shutdown({bool immediate = false}) async {
+    final request = Request(
+        'POST',
+        Uri.http('localhost', 'shutdown',
+            {'mode': '"POWER_OFF"', 'immediate': '$immediate'}));
```
Comment (dbungert) [inline packages/subiquity_client/lib/subiquity_client.dart:246]: The final mode term got changed to "POWEROFF" instead of having the underscore.

**Review by dbungert — CHANGES_REQUESTED** [review]

File: packages/subiquity_client/lib/subiquity_client.dart (hunk 1 above)
Comment (jpnurmi) [reply packages/subiquity_client/lib/subiquity_client.dart]: Thanks

**Comment by oSoMoN** [comment]
Shouldn't this PR also update the subiquity checkout commit ID, now that https://github.com/canonical/subiquity/pull/1029 has landed?

**Comment by jpnurmi** [comment]
subiquity was already updated: https://github.com/canonical/ubuntu-desktop-installer/pull/249

**Comment by oSoMoN** [comment]
Oh right, I missed that. LGTM then.

**Pull request merged**